src/components/Preloader.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { css } from "@emotion/core";
+import MoonLoader from "react-spinners/MoonLoader";
+
+// Adding custom styles
+const override = css`
+    display: flex;
+    margin: 15.7vh auto;
+    border-color: red;
+    z-index: 9999;
+`;
+
+
+class Preloader extends React.Component {
```

Comment:
Is this reloader from the design? I feel like this might be too over-whelming, shall we considering using a smaller & simpler spinner?

Reply:
Fixed it